sql/moz-fx-data-shared-prod/firefox_accounts/docker_fxa_customs_sanitized/view.sql

```diff
@@ -2,6 +2,105 @@ CREATE OR REPLACE VIEW
   `moz-fx-data-shared-prod.firefox_accounts.docker_fxa_customs_sanitized`
 AS
 SELECT
-  *
+  `date`,
+  logName,
+  (
+    SELECT AS STRUCT
+      resource.type,
+      (
+        SELECT AS STRUCT
+          resource.labels.instance_id,
+          resource.labels.zone,
+          resource.labels.project_id,
+          CAST(NULL AS STRING) AS cluster_name,
+          CAST(NULL AS STRING) AS namespace_name,
+          CAST(NULL AS STRING) AS container_name,
+          CAST(NULL AS STRING) AS pod_name,
+          CAST(NULL AS STRING) AS location
+      ) AS labels
+  ) AS resource,
+  textPayload,
+  (
+    SELECT AS STRUCT
+      jsonPayload.block,
+      jsonPayload.op,
+      jsonPayload.action
+  ) AS jsonPayload,
+  `timestamp`,
+  receiveTimestamp,
+  severity,
+  insertId,
+  httpRequest,
+  (
+    SELECT AS STRUCT
+      labels.compute_googleapis_com_resource_name,
+      CAST(NULL AS STRING) AS k8s_pod_deployment,
+      CAST(NULL AS STRING) AS k8s_pod_app_kubernetes_io_component,
+      CAST(NULL AS STRING) AS k8s_pod_env_code,
+      CAST(NULL AS STRING) AS k8s_pod_pod_template_hash,
+      CAST(NULL AS STRING) AS k8s_pod_app_kubernetes_io_name,
+      CAST(NULL AS STRING) AS k8s_pod_app_kubernetes_io_instance,
+  ) AS labels,
```

Comment:
I suspect they don't really matter, but the old table has some additional `labels` fields: `application`, `stack`, `env`, `type`.